protocol/src/test/java/org/opensearch/sql/protocol/response/format/RawResponseFormatterTest.java

```diff
@@ -153,7 +153,12 @@ void replaceNullValues() {
         + "John|Seattle%n"
         + "|Seattle%n"
         + "John|";
-    assertEquals(format(expected), rawFormater.format(response));
+    assertEquals(format(expected), rawFormatter.format(response));
+  }
+
+  @Test
+  void testContentType() {
+    assertEquals(rawFormatter.contentType(), "plain/text; charset=UTF-8");
```

Comment:
if you use CONTENT_TYPE here, then your tests will pass even if the header text changes.

Reply:
implemented in https://github.com/Bit-Quill/opensearch-project-sql/pull/279/commits/1fce3446613b66bee8476b87f8d8a5f5da7bb66e